### ISSUE TITLE
code not opening in browser

### ISSUE THREAD

**Issue by nfolkman** [microsoft/vscode#141177]
Issue Type: <b>Feature Request</b>

Error:
"This site can’t be reachedlocalhost refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED"

I'm new to VSCode and am not sure where to go from here. I've uninstalled and reinstalled the program multipe times, uninstalled and reinstalled Chrome, tested various code, and the response is the same.

VS Code version: Code 1.63.2 (899d46d82c4c95423fb7e10e68eba52050e30ba3, 2021-12-15T09:40:02.816Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No

**Comment by nfolkman** [comment]
If you could let me know what info exactly will be needed to solve this type of issue, I would be happy to provide it.

**Comment by jrieken** [comment]
Please describe in steps what you are trying to achieve

**Comment by nfolkman** [comment]
> Please describe in steps what you are trying to achieve

I was designing an html page with VSCode, and initially I was able to run the code and open the page in Google Chrome. However, later that day I tried to run the code again and an error message pulled up in the browser saying:

"This site can’t be reached localhost refused to connect.
Try:

Checking the connection
Checking the proxy and the firewall
ERR_CONNECTION_REFUSED"



I'm new to VSCode and am not sure where to go from here in order to fix this issue. I've uninstalled and reinstalled the program multipe times, uninstalled and reinstalled Chrome, tested various code files, and the response is the same.

Is there a setting, or selection of possibilities that might be causing this issue with running the code in a browser?
